distributionviewer/core/static/js/app/components/views/chart.js

```diff
@@ -102,10 +50,102 @@ export class Chart extends React.Component {
       return chart;
     }
   }
+
+  setPointsMetaState(showOutliers) {
+    const pointsMetaLength = this.pointsMeta.length;
+
+    /* eslint-disable camelcase */
+    if (showOutliers) {
+      this.setState({
+        min_x: this.pointsMeta[0].x,
+        max_x: this.pointsMeta[pointsMetaLength - 1].x,
+      });
+    } else {
+      this.setState({
+        min_x: this.pointsMeta[Math.max(Math.round(pointsMetaLength * .005) - 1, 0)].x,
+        max_x: this.pointsMeta[Math.min(Math.round(pointsMetaLength * .995) - 1, pointsMetaLength)].x,
```

Comment:
The math here is fishy for sure. My understanding is we'll need to figure out the standard deviation and decide to drop any values that exceed it (though technically our request is to use the central 99%). Once we pop those from the array we wouldn't need to mess with min_x, max_x etc.

If the idea really is to just use the "central 99%" we'll need to sort the values in the data (only if it exceeds 99 entries) and then drop the head and tail ends as needed. I can recommend a brand of bourbon that might help with these functions.